groclient/lib.py

```diff
@@ -488,13 +489,15 @@ def list_of_series_to_single_series(series_list, add_belongs_to=False, include_h
                 'end_date': point[1],
                 'value': point[2],
                 'unit_id': point[4] if len(point) > 4 else series['series'].get('unitId', None),
-                'metadata': point[5] if len(point) > 5 else {},
+                'metadata': point[5] if len(point) > 5 and point[5]!=None else {},
```

Comment:
nit: `is not None` is preferred (2nd point here: https://www.python.org/dev/peps/pep-0008/#programming-recommendations)

For older clients, will they have a `metadata` value of `None` instead of `{}` after the API server change?

Reply:
For the second question, clients on older version will still get {}. Users will not be able to turn on the showAvailableDate option if they don't upgrade to the newer version, so availableDate is always null. Since we are removing trailing Nulls in each data point(dataUtils - flattenTimeSeriesPoint()), the API response becomes like this:
1) if metadata != Null
[{series: {...}, data: [[start, end, value, reportingDate, unit, metadata], ...]}]
2) if metadata = Null, it will be deleted together with availableDate
[{series: {...}, data: [[start, end, value, reportingDate, unit], ...]}]

Thus for current version, the statement _len(point) > 5_ is equivalent to _metadata is not None_